csharp-tree-sitter.el

```diff
@@ -307,7 +310,12 @@
     (outdent . ;; these nodes always outdent (1 shift in opposite direction)
              (;; "}"
               case_switch_label
+
               ))
+
+    (align-to-node-line . ;; these nodes are aligned to the first column of the
```

Comment:
Can you try to reword this comment? I've read it a couple of times now, and I don't really understand what it means 😇

Reply:
I tried again. This one is a hard one to explain without going step by step of the algorithm (what I basically ended up doing .-.). Hope it is clear, now. :)

Reply:
This is a little clearer! Can I ask you to align the comments a little more neatly? I think I understood what was happening, but I guess I won't in a months time or so :)